hypothesis-python/docs/tutorial/examples.rst

```diff
@@ -0,0 +1,17 @@
+Explicit examples
+=================
+
+If you want Hypothesis to always run a specific input, you can use the |@example| decorator. |@example| adds an explicit input which Hypothesis will run every time, in addition to the randomly generated examples. You can think of explicit examples as combining unit-testing with property-based testing.
+
+For instance, suppose we are testing something using integers, but want to make sure we try a few large prime numbers on every test run. We can add these primes with an explicit |@example|:
+
+.. code-block:: python
+
+    # mersenne primes
+    @example(2**17 - 1)
+    @example(2**19 - 1)
+    @given(st.integers())
+    def test_something_with_integers(n):
+        pass
+
+Hypothesis runs all explicit examples first, in the |Phase.explicit| phase, before generating additional random examples in the |Phase.generate| phase.
```

Comment:
Should we mention that explicit examples do not shrink? Perhaps referencing https://stackoverflow.com/questions/77817658/can-hypothesis-shrink-an-failing-test-case-which-was-not-found-by-hypothesis